docs/design/services/dynamodb/high-level-library/prototype/option-1/sync/Prototype.java

```diff
@@ -0,0 +1,1025 @@
+/**
```

Comment:
what would be the package name?

Reply:
It's up to debate, but I was thinking a group ID and package name of software.amazon.aws. The target libraries aren't necessarily always going to be under the AWS SDK umbrella (eg. KCL could be one example), so a general ".aws" felt appropriate.